dashboard/src/components/api/codenvy-machine-auth.factory.js

```diff
@@ -0,0 +1,66 @@
+/*
+ *  [2015] - [2016] Codenvy, S.A.
+ *  All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Codenvy S.A. and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Codenvy S.A.
+ * and its suppliers and may be covered by U.S. and Foreign Patents,
+ * patents in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Codenvy S.A..
+ */
+'use strict';
+
+export class CodenvyMachineAuth {
+
+         /**
+          * Default constructor that is using resource
+          * @ngInject for Dependency injection
+          */
+         constructor($injector, $q) {
+           this.$injector = $window;
```

Comment:
I think there is an issue there ($window is not provided in constructor parameter)